lib/semian/circuit_breaker.rb

```diff
@@ -26,6 +27,11 @@ def initialize(name, exceptions:, success_threshold:, error_threshold:,
       @error_timeout = error_timeout
       @exceptions = exceptions
       @half_open_resource_timeout = half_open_resource_timeout
+      @lumping_interval = lumping_interval
+
+      if @lumping_interval >= @error_threshold_timeout && @lumping_interval > 0
```

Comment:
do we need the `&&` here? I think what would make more sense is to prevent negative values for all parameters instead